app/src/main/res/layout-land/topic_info_fragment.xml

```diff
@@ -20,14 +20,14 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:paddingTop="@dimen/padding_24dp"
-      android:paddingBottom="24dp">
+      android:paddingBottom="@dimen/padding_24dp">
 
       <TextView
         android:id="@+id/topic_name_text_view"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginStart="32dp"
-        android:layout_marginEnd="32dp"
+        android:layout_marginStart="72dp"
```

Comment:
Add margin top to this textview.

Reply:
@veena14cs  marginTop was provided in mock of 32dp but 32dp is making  a lot of unnecessary space in between actionBar and Topic name 

screenshot if marginTop is 32dp
![pf32](https://user-images.githubusercontent.com/44704035/76205315-fb6db280-621f-11ea-9529-270776c45f47.jpeg)

screenshot without any margin
![Al](https://user-images.githubusercontent.com/44704035/76140175-a98d2700-607d-11ea-92fc-8b784c727c8d.jpeg)

Reply:
You have given 24dp padding to constraint layout. You can give top margin of 8dp for the textview so that total will result in 32dp.

Reply:
update marginTop with 8dp in my most Recent pr 
@veena14cs  thanks